pypeit/spectrographs/spectrograph.py

```diff
@@ -553,45 +579,53 @@ def get_meta_value(self, inp, meta_key, required=False, ignore_bad_header=False,
                         self.meta[meta_key]['card']))
             return None
 
+        # JFH Old code which causes a crash when the type is wrong
         # Deal with dtype (DO THIS HERE OR IN METADATA?  I'M TORN)
-        if self.meta_data_model[meta_key]['dtype'] == str:
-            value = str(value).strip()
-        elif self.meta_data_model[meta_key]['dtype'] == int:
-            value = int(value)
-        elif self.meta_data_model[meta_key]['dtype'] == float:
-            value = float(value)
-        elif self.meta_data_model[meta_key]['dtype'] == tuple:
-            assert isinstance(value, tuple)
-        else:
-            debugger.set_trace()
+        #if self.meta_data_model[meta_key]['dtype'] == str:
```

Comment:
remove?

Reply:
Still debating with X about how to proceed so I'll leave it in.